src/bun.js/api/BunObject.zig

```diff
@@ -393,14 +404,25 @@ pub fn shellParse(
     defer arena.deinit();
 
     const template_args = callframe.argumentsPtr()[1..callframe.argumentsCount()];
+    var stack_alloc = std.heap.stackFallback(@sizeOf(bun.String) * 4, arena.allocator());
```

Comment:
i wonder if a helper function for this abstraction would be ncie
ive seen it a lot "please stack fallback for <n> items but it technically can be infinity so heap in that case"